### ISSUE TITLE
gl init crashes with “fatal: not a git repository”

### ISSUE THREAD

**Issue by loewenheim** [sdg-mit/gitless#183]
When I run `gl init` in an empty directory, I get
```
fatal: not a git repository (or any of the parent directories): .git
```

When I run it in a directory where a git repo already exists, I get:
```
fatal: 'init' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights and the repository exists.
```

My os is nixos. Both git 2.18.0 and gitless 0.8.6 are installed from the `unstable` channel.

**Comment by spderosso** [comment]
Thank you for reporting this. This problem is tracked in #179. The fix introduced in #180 should have fixed it. I will cut a new release with the fix soon.

**Comment by spderosso** [comment]
Duplicate of #179
